docs/creating-test-environment.md

```diff
@@ -1,315 +1,198 @@
+1. First, install the module template in the control-plane to make it available for all Kyma installations:
 
-## Start the Installation
+   In `samples/template-operator`, run
 
-_Note for Remote Clusters: Make sure you run the commands with `KUBECONFIG` set to the KCP Cluster_
+    ```sh
+    make module-template-push
+    ```
 
-First, install the module template in the control-plane to make it available for all Kyma installations:
+   to apply the module-template.
 
-In `samples/template-operator`, run
 
-```sh
-make module-template-push
-```
+2. Create a request for kyma installation of the module in `samples/template-operator` of the lifecycle-manager with
 
-to apply the module-template
+    ```sh
+    sh hack/gen-kyma.sh
+    kubectl apply -f kyma.yaml
+    ```
 
-Create a request for kyma installation of the module in `samples/template-operator` of the lifecycle-manager with
+3. Now try to check your kyma installation progress, e.g. with `kubectl get kyma -n kyma-system -ojsonpath={".items[0].status"} | yq -P`:
 
-```sh
-sh hack/gen-kyma.sh
-kubectl apply -f kyma.yaml
-```
+    ```yaml
+    conditions:
+      - lastTransitionTime: "2022-08-18T18:10:09Z"
+        message: module is Ready
+        reason: template
+        status: "True"
+        type: Ready
+    moduleInfos:
+      - moduleName: template
+        name: templatekyma-sample
+        namespace: kyma-system
+        templateInfo:
+          channel: stable
+          generation: 1
+          gvk:
+            group: operator.kyma-project.io
+            kind: Manifest
+            version: v1alpha1
+    observedGeneration: 1
+    state: Ready
+    ```
 
-Now try to check your kyma installation progress, e.g. with `kubectl get kyma -n kyma-system -ojsonpath={".items[0].status"} | yq -P`:
-
-```yaml
-conditions:
-  - lastTransitionTime: "2022-08-18T18:10:09Z"
-    message: module is Ready
-    reason: template
-    status: "True"
-    type: Ready
-moduleInfos:
-  - moduleName: template
-    name: templatekyma-sample
-    namespace: kyma-system
-    templateInfo:
-      channel: stable
-      generation: 1
-      gvk:
-        group: operator.kyma-project.io
-        kind: Manifest
-        version: v1alpha1
-observedGeneration: 1
-state: Ready
-```
+### 3.4.1 Verify the installation
 
-Also, you can observe the installation in the runtime by switching the context to the SKR context and then verifying the status.
+You can observe the installation in the runtime by switching the context to the SKR context and then verifying the status.
 
-`kubectl config use-context k3d-op-skr && kubectl get samples.operator.kyma-project.io -n kyma-system -ojsonpath={".items[0].status"} | yq -P`
+`kubectl get kymas.operator.kyma-project.io -n kyma-system -ojsonpath={".items[0].status"} | yq -P`
```

Comment:
This was originaly meant to be used inside SKR so this command was trying to fetch the module state (not kyma state). Fetching the remote kyma state is done with
`kubectl get kyma -n kyma-system -ojsonpath={".items[0].status"} | yq -P`